pkg/app/log.go

```diff
@@ -76,3 +88,18 @@ func setLogLevel(logLevel string) {
 		log.SetLevel(log.InfoLevel)
 	}
 }
+
+type ProxyJsonWrapperFormatter struct {
+	WrappedFormatter log.Formatter
+}
+
+func (f *ProxyJsonWrapperFormatter) Format(entry *log.Entry) ([]byte, error) {
+	// if proxying the json message is intended, just return the bytes
+	// TODO: Find a more elegant way to carry this info
+	if entry.Data[ProxyJsonLogKey] == true {
+		return []byte(entry.Message), nil
```

Comment:
```suggestion
		b := bytes.NewBufferString(entry.Message)
		b.WriteString("\n")
		return b.Bytes(), nil
```

I've tried this new version and log lines are glued together because `json.Marshal` returns buffer without `\n`. The original JSONFormatter uses `NewEncoder.Encode` which adds `\n`, but adding the newline here is more simpler solution.